Dockerfile: cannot load module pkg/promlib listed in go.work file

### What happened?

Same as #83362, but with this error:

```
 => ERROR [go-builder 10/24] RUN go mod download                                                  0.6s
------                                                                                                 
 > [go-builder 10/24] RUN go mod download:
0.551 go: cannot load module pkg/promlib listed in go.work file: open pkg/promlib/go.mod: no such file or directory
------
Dockerfile:61
--------------------
  59 |     COPY pkg/apimachinery/go.* pkg/apimachinery/
  60 |     
  61 | >>> RUN go mod download
  62 |     RUN if [[ "$BINGO" = "true" ]]; then \
  63 |           go install github.com/bwplotka/bingo@latest && \
--------------------
ERROR: failed to solve: process "/bin/sh -c go mod download" did not complete successfully: exit code: 1
make: *** [Makefile:236 : build-docker-full] Error 1
```

### What did you expect to happen?

Image building without error

### Did this work before?

yes

### How do we reproduce it?

```shell
make build-docker-full
```

### Is the bug inside a dashboard panel?

_No response_

### Environment (with versions)?

Grafana: 9b2058f814b5f6e56598e57c5dd427f2edf033c4
OS:
Browser:


### Grafana platform?

Docker

### Datasource(s)?

_No response_

--- a/Dockerfile
+++ b/Dockerfile
@@ -57,6 +57,7 @@ COPY .bingo .bingo
 COPY pkg/util/xorm/go.* pkg/util/xorm/
 COPY pkg/apiserver/go.* pkg/apiserver/
 COPY pkg/apimachinery/go.* pkg/apimachinery/
+COPY pkg/promlib/go.* pkg/promlib/
 
 RUN go mod download
 RUN if [[ "$BINGO" = "true" ]]; then \

